src/floating_label.cpp

```diff
@@ -189,7 +189,8 @@ void floating_label::undraw()
 void floating_label::update(int time)
 {
 	if(!create_texture()) {
-		ERR_FT << "failed to create texture for floating label" << std::endl;
+		// Can't display this error because WML tests die...
+		//ERR_FT << "failed to create texture for floating label" << std::endl;
```

Comment:
Is there a way to check if the display is faked and skip creating the texture in that case? WML tests are run with implicit `--nogui` so they shouldn't be rendering anything anyway.

Reply:
technically yes...